test/integration/dialects/postgres/query-generator.test.js

```diff
@@ -870,7 +873,7 @@ if (dialect.match(/^postgres/)) {
           expectation: 'CREATE INDEX \"bar\" ON \"User\" (\"username\", \"isAdmin\")'
         }, {
           arguments: ['mySchema.User', ['username', 'isAdmin'], {}, 'User'],
```

Comment:
True but when using migrations it will be much cleaner to type `schema.table` than `{schema: schema, table: table}`